packages/dev-server/src/MetroDevServer.ts

```diff
@@ -1,15 +1,35 @@
-import { getConfig, projectHasModule } from '@expo/config';
+import http from 'http';
+import { Platform, getConfig, projectHasModule } from '@expo/config';
 import { createDevServerMiddleware } from '@react-native-community/cli-server-api';
 import Log from '@expo/bunyan';
 import * as ExpoMetroConfig from '@expo/metro-config';
 import bodyParser from 'body-parser';
+import type Metro from 'metro';
```

Comment:
I wasn't aware you could do this in TypeScript.

Reply:
It was added in TS 3.8. I guess it’s not needed very often, but here it’s invaluable 👍